### DISCUSSION TITLE
Disable touch command

### DISCUSSION THREAD

**Discussion by Sonusss** [HASwitchPlate/openHASP#519] [Ideas]
Hi,

Having a command to disable the touchscreen for cleaning purpose would be meaningful...
Every time my wife is trying to clean my plates the house is getting crazy.
The only turn around I have by now is manually disabling integration in HA but unfortunately it can't be done from a script.   
Having this command would allow a script disabling all plates for cleaning.
OR did I miss something ?

CHEERS

**Comment by fvanroie** [comment]
You can create an object on page 0, make it as large as the display to catch the touches without affecting anything.
Give it a recognizable id and you can remove the object from the plate(s) once the cleaning is done.
If you give the catch object an obvious color, like red, it should be visible if the plate is active or not...

**Reply by Sonusss** [reply]
I tried this option which works indeed but I feel this solution incomplete as the screen still react on touch making the cleaning person think being doing something wrong...  I keep on thinking that having a command to disable the touch function is a plus and a combination of both would be perfect in this case. However having the option to disable touch catch might be helpful in other cases too....

**Reply by htvekov** [reply]
If device is idle then screen will indeed react on touch and turn on BL and set brightness. But only mqtt events for your dummy page 0 object will be sent.
If you want device to wake up and stay at same brightness level throughout the cleaning process, then you could issue an `idle_off` command and also set the idle timers to zero issuing `config/gui {"idle1":0,"idle2":0}` when you turn on your dummy page 0 cleaning object.
You'll have to reset the idle1 & 2 values when cleaning is done though
